run_meson_command_tests.py

```diff
@@ -63,15 +63,14 @@ def tearDown(self):
 
     def _run(self, command, workdir=None):
         '''
-        Run a command while printing the stdout and stderr to stdout,
-        and also return a copy of it
+        Run a command while printing the stdout, and also return a copy of it
         '''
         # If this call hangs CI will just abort. It is very hard to distinguish
         # between CI issue and test bug in that case. Set timeout and fail loud
         # instead.
         p = subprocess.run(command, stdout=subprocess.PIPE,
-                           stderr=subprocess.STDOUT, env=os.environ.copy(),
-                           universal_newlines=True, cwd=workdir, timeout=60 * 5)
```

Comment:
Why did you change this?

Reply:
See https://github.com/mesonbuild/meson/pull/4349#issuecomment-433273101 or the commit message for explanations of the changes:
> `run_meson_commands.py` was confusing the warning message about compdb as the `meson command is ...` line, so I changed `CommandTests._run` to only capture stdout.